src/configuration-manager.ts

```diff
@@ -116,6 +116,14 @@ export class ConfigurationManager {
         await this.tspClient?.request(CommandTypes.Configure, args);
     }
 
+    public async configurePlugin(pluginName: string, configuration: unknown): Promise<void> {
+        const args: tsp.ConfigurePluginRequestArguments = {
+            configuration,
+            pluginName,
+        };
+        await this.tspClient?.request(CommandTypes.ConfigurePlugin, args);
```

Comment:
You could follow vscode a bit more closely:

 - use `executeWithoutWaitingForResponse`
 - add API version check
 - make the function sync

https://github.com/microsoft/vscode/blob/97c8a97261377a11dc5d6b547e275f4cb4e9399a/extensions/typescript-language-features/src/typescriptServiceClient.ts#L1007-L1012

Reply:
And to be fair, since it becomes a one-liner and it's only used from one place, I would just inline it where the command is handled.

Reply:
Thanks for the feedback, I've updated per your suggestions.